apps/editing-toolkit/editing-toolkit-plugin/tags-education/src/add-tags-education-link.js

```diff
@@ -0,0 +1,45 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useLocalizeUrl } from '@automattic/i18n-utils';
+import { ExternalLink } from '@wordpress/components';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { addFilter } from '@wordpress/hooks';
+import { __ } from '@wordpress/i18n';
+
+const trackTagsEducationLinkClick = () =>
+	recordTracksEvent( 'wpcom_block_editor_tags_education_link_click' );
+
+const addTagsEducationLink = createHigherOrderComponent( ( PostTaxonomyType ) => {
+	return ( props ) => {
+		const localizeUrl = useLocalizeUrl();
+
+		if ( props.slug !== 'post_tag' ) {
+			return <PostTaxonomyType { ...props } />;
+		}
+
+		return (
+			<>
+				<PostTaxonomyType { ...props } />
+				<ExternalLink
+					href={ localizeUrl(
+						'https://wordpress.com/support/posts/tags',
+						// TODO: remove tagsEducationLocale after fixing useLocalizeUrl.
```

Comment:
Could we link to #55527 in this comment please?
Just so it's clear what we're waiting on